### DISCUSSION TITLE
Documento PDF notifica

### DISCUSSION THREAD

**Discussion by Hdemografici** [italia/ansc#1118] [Supporto servizi cooperativi] [answered]
buon pomeriggio

@asqueglia collegandoci a quanto chiesto nella issue #1091 si chiede se, nell'ambito dei servizi cooperativi, esiste un servizio che permetta di scaricare il documento PDF collegato alle notifiche pervenute ad un determinato Comune (in WA bottone "Visualizza PDF" presente in corrispondenza della singola notifica)

grazie per l'attenzione

Halley informatica

**Comment by mttfranci** [comment, marked as answer]
Salve @Hdemografici dovreste poter invocare il servizio di anteprima usando l'id anteprima cone id evento.

Ad esempio se la notifiva ha id : 2024-123456-00000-123123, potete utilizzare il path : 

/anteprima/evento/1

Indicando nella sezione evento : 

    "evento": {
        "id": "123456",
        "idTipoEvento": "1",
        "idAnsc": "2024-123456-00000-123123"
    }  

Saluti

Assistenza ANSC
